stdlib/http/src/main/java/org/ballerinalang/net/http/WebSocketDispatcher.java

```diff
@@ -107,7 +107,10 @@ static WebSocketService findService(WebSocketServicesRegistry servicesRegistry,
         } catch (BallerinaConnectorException | URITemplateException e) {
             String message = "No Service found to handle the service request";
             webSocketHandshaker.cancelHandshake(404, message);
-            throw new BallerinaConnectorException(message, e);
+            if (log.isDebugEnabled()) {
+                log.debug(message, e);
+            }
```

Comment:
Shall we make the logger level to error? IMO we need to pass the error to the user. WDYT?